portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro

```diff
@@ -468,6 +514,14 @@ definition {
 			expected = "${expectedTotalElement}");
 	}
 
+	macro assertResponseHasIdFieldValueOnly {
```

Comment:
I see this one being the exact same with `assertResponseHasTitleFieldValueOnly` Maybe what we could do here is to pass the value of items and do a direct assertion in a `.testcase` layer?
In macro we could then have:

```
macro getItemsDetail { 
var itemsDetail = JSONUtil.getWithJSONPath("${responseToParse}", "$.items");
return "${response}";}

```
And in .testcase it could be something like:

```
task ("Then in a response I receive a id values only") {
var actualValue = HeadlessWebcontentAPI.getItemsDetail(responseToParse = "${responseToParse}")
TestUtils.assertEquals(
			actual = "${actualValue}",
			expected = {id=${structuredContentId}}");
		} 
```
The reason to propose this is that, if we can make the assertion on the .testcase level it is always better (more readable for the test), unless we do various checks in one assertion (like we sometimes do with Headless tests). Let me know what do you think about it! :)